_opcodes/op163-bgee.html

```diff
@@ -0,0 +1,26 @@
+---
+n: 163
+name: "Protection: Free Action"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies the free action effect to the targeted creature(s), restoring the creatures movement rate to its default value.<br/>
+In particular, it only removes <a href="../_opcodes/op126.html">opcode #126</a> effects using 'param2 = 1' (Set value), if the creature's current movement rate is less than the default movement rate for its current animation (Haste/Slow are not factored in this).
+
+{% capture note %}
+Does NOT remove "Increment" or "Set % Value" effects, nor any using the alternate movement rate opcode, <a href="../_opcodes/op176.html">opcode #176</a>.
+{% endcapture %}
+
+{% include important.html %}
+
+{% capture note %}
+Similarly affects <a href="../_opcodes/op177.html">opcode #177</a>/<a href="../_opcodes/op283-bgee.html">opcode #283</a> effects if the <a href="../file_formats/ie_formats/eff_v2.htm">EFF</a> file is using the above values.
+{% endcapture %}
+
+{% include note.html %}
```

Comment:
This is likely also true for the other games, so please add a note to the main one. Eg. "Also check out the EE description here."

Reply:
Done.